docs/src/main/sphinx/object-storage/legacy-azure.md

```diff
@@ -192,3 +192,59 @@ CREATE TABLE hive.abfs_export.orders_abfs
 WITH (format = 'ORC')
 AS SELECT * FROM tpch.sf1.orders;
 ```
+
+(legacy-azure-migration)=
+## Migration to native Azure implementation
+
+Trino includes a {doc}`native Azure implementation
+</object-storage/file-system-azure>` that does not rely on the legacy Hadoop
+implementation. Trino recommends upgrading existing deployments to the new
+native implementation at your earliest convenience.
+
+The following considerations apply to legacy configurations moving to the
+native implementation:
+
+* Legacy storage types (WASB and ADL) are not supported.
+* Azure Government is not supported.
+
+To migrate a catalog to use the native filesystem implementation for Azure, make
+the following edits to your catalog configuration:
+
+1. Add the `fs.native-azure.enabled=true` catalog configuration property.
+2. Add the `azure.auth-type` catalog configuration property. Set to `ACCESS_KEY`
+   for access key authentication or `OAUTH` for OAuth 2.0 authentication. Leave
+   unconfigured for [default
+   authentication](https://learn.microsoft.com/java/api/overview/azure/identity-readme?view=azure-java-stable#defaultazurecredential).
+3. Refer to the following table to rename your existing legacy catalog
+   configuration properties to the corresponding native configuration
+   properties. Supported configuration values are identical unless otherwise
+   noted.
+
+  :::{list-table}
+  :widths: 35, 35, 65
+  :header-rows: 1
+   * - Legacy property
+     - Native property
+     - Notes
+   * - `hive.azure.abfs-access-key`
+     - `azure.access-key`
+     -
+   * - `hive.azure.abfs.oauth.endpoint`
+     - `azure.oauth.endpoint`
+     - Also see `azure.oauth.tenant-id` in {ref}`azure-oauth-authentication`.
+   * - `hive.azure.abfs.oauth.client-id`
+     - `azure.oauth.client-id`
+     -
+   * - `hive.azure.abfs.oauth.secret`
+     - `azure.oauth.secret`
+     -
+   * - `hive.azure.abfs.oauth2.passthrough`
+     - `azure.use-oauth-passthrough-token`
+     -
+  :::
+
+4. Remove the following legacy configuration properties if they exist in your
+   catalog configuration:
+
+      * `hive.azure.abfs-storage-account`
+      * `hive.azure.wasb*`
```

Comment:
Reword to a text like "Add properties starting with `hive.azure.wasb`" or list them

Reply:
Suggest:
* All properties beginning with `hive.azure.wasb`

Reply:
I'll just list them, I only found two